test/NuGet.Core.Tests/NuGet.Frameworks.Test/NuGetFrameworkParseTests.cs

```diff
@@ -224,6 +225,13 @@ public void NuGetFramework_ProfileName(string folder, string expected)
         [InlineData("netcoreapp1.5", ".NETCoreApp,Version=v1.5")]
         [InlineData("netcoreapp2.0", ".NetCoreApp,Version=v2.0")]
         [InlineData("netcoreapp3.0", ".NetCoreApp,Version=v3.0")]
+        [InlineData("net5.0-android", ".NETCoreApp,Version=v5.0,Platform=Android")]
```

Comment:
I think I'd expect PlatformVersion=v0.0 similar to what "netcoreapp" would do for the framework version. 

Platform and Platform version *always* go together.

Reply:
actually...this is reversed from most of our tests...so the input is "Platform=Android" haha. 

I'd also test PlatformVersion=v0.0 in that case.

Reply:
obsolete -- this is no longer the format.